packages/server/tsconfig.json

```diff
@@ -3,6 +3,6 @@
   "include": ["src", "test"],
   "compilerOptions": {
     "target": "ES2019",
-    "lib": ["ES2020.Promise"]
+    "lib": ["es5", "es6", "dom", "dom.iterable", "ES2020.Promise"]
```

Comment:
Might revisit this later. I want to do next major as node > 14, so don't think all of it is needed